pkg/executor/docker/handler.go

```diff
@@ -101,6 +101,23 @@ func (h *executionHandler) run(ctx context.Context) {
 	case exitStatus := <-statusCh:
 		// success case, the container completed its execution, but may have experienced an error, we will attempt to collect logs.
 		containerExitStatusCode = exitStatus.StatusCode
+		containerJSON, err := h.client.ContainerInspect(ctx, h.containerID)
+		if err != nil {
+			h.logger.Warn().Err(err).Msg("failed to inspect docker container")
+			h.result = &models.RunCommandResult{
+				ExitCode: int(containerExitStatusCode),
+				ErrorMsg: err.Error(),
+			}
+			return
+		}
+		if containerJSON.ContainerJSONBase.State.OOMKilled {
+			containerError = errors.New(`memory limit exceeded. Please refer to https://docs.bacalhau.org/getting-started/resources/#docker-executor for more information`)
+			h.result = &models.RunCommandResult{
+				ExitCode: int(containerExitStatusCode),
+				ErrorMsg: fmt.Sprintf(containerError.Error()),
```

Comment:
You don't need fmt.Sprintf() here as containerError.Error() is already a string. This is the cause of the linter errors you're seeing.